src/test/java/hw5/steps/ActionStep.java

```diff
@@ -0,0 +1,69 @@
+package hw5.steps;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.When;
+import java.util.List;
+import org.openqa.selenium.WebElement;
+
+public class ActionStep extends AbstractStep {
+
+    @Given("I open JDI GitHub site")
+    public void openHomePage() {
+        homePage.open(BASE_URL);
+    }
+
+    @When("I login as user {string}")
+    public void performLogin(String user) {
+        homePage.getHeader().signIn(properties.getString("login"), properties.getString("pass"));
+    }
+
+    @When("I click on {string} button in Header")
+    public void clickOnButtonInHeader(String buttonName) {
+        homePage.getHeader().clickOnHeaderElement(buttonName);
+    }
+
+    @When("I click on {string} button in Service dropdown")
+    public void clickOmButtonInServiceDropdown(String buttonName) {
+        homePage.getHeader().clickOnOptionInDropdownService(buttonName);
+    }
+
+    @When("I select checkboxes")
+    public void selectCheckboxes(List<String> checkboxNames) {
+        checkboxNames.forEach(checkbox ->
+            differentElementsPage
+                .getDifferentElements()
+                .clickOnCheckbox(checkbox));
+    }
+
+    @When("I select radio button {string}")
+    public void selectRadio(String radio) {
+        differentElementsPage
+            .getDifferentElements()
+            .clickOnRadioButton(radio);
+    }
+
+    @When("I select dropdown {string}")
+    public void selectInDropdown(String dropdown) {
+        differentElementsPage
+            .getDifferentElements()
+            .getDropdown(dropdown);
+    }
+
+    @When("I select 'vip' checkbox for {string}")
+    public void selectVipCheckboxForUser(String username) {
+        userTablePage.getUserTable().getUserTableWithReplace().forEach(line -> {
+            if (line.get(1).equals(username)) {
+                userTablePage
+                    .getUserTable()
+                    .clickOnCheckboxForUser(Integer.parseInt(line.get(0)));
+
+                if (!userTablePage.getUserTable().getCheckboxForUser(Integer.parseInt(line.get(0))).isSelected()) {
+                    userTablePage
+                        .getUserTable()
+                        .clickOnCheckboxForUser(Integer.parseInt(line.get(0)));
+                }
+            }
+            }
+        );
```

Comment:
It looks like this method selects checkbox for a wrong user. `line.get()` for user `Sergey Ivan` returns `2` (NumberType column value), but `clickOnCheckboxForUser(2)` selects checkbox for user `Vladzimir` because numbering for checkbox elements starts with 0.
And I am not sure that this `if` is needed:
```java
if (!userTablePage.getUserTable().getCheckboxForUser(Integer.parseInt(line.get(0))).isSelected())
```

Reply:
block "if" deleted, method fixed

Reply:
The method has not been changed. Are you sure the changes have been committed? 
<img width="691" alt="image" src="https://user-images.githubusercontent.com/46869924/144565275-ab50befe-9d49-4c2a-90fe-0525ca314e70.png">